archlinuxcn/clash-verge/PKGBUILD

```diff
@@ -32,7 +32,7 @@ source=("${pkgname}-${pkgver}.tar.gz::${url}/archive/refs/tags/v${pkgver}.tar.gz
 )
 sha512sums=('9fe6676545d63768b156270d17e1bf75fef6c3d27202962faed5016d22fd237f6f223e7a917a8bd4fc6d200ea4db7614a687c930d478bdd02c1f7b0fb058ae6f'
             '2066dacf2e5e0135e6403cbfb825efcdf08bbcdc781407e6bb1fbb85143817b2b1abef641d20390ff7e5b3e91a509933e9eb17a64f9de7671445ac6d5363a44a'
-            'd1e8992cc0de168930002adccf7953416dfac199ea11886043de58b0df366e4337d5ec548fa3bf8be30575596a5e71dd31960db72f9e067ff131feb756bc96c1')
```

Comment:
```
==> Validating source files with sha512sums...
    clash-verge-1.4.10.tar.gz ... Passed
    clash-verge.desktop ... Passed
    001-disable-updater.patch ... FAILED
==> ERROR: One or more files did not pass the validity check!
==> ERROR: Could not download sources.
```

Reply:
em.....
i updated patch, so i changed sums here. I believe the new sums is right.(tried just now, and fine)